core/host_api/impl/crypto_extension.cpp

```diff
@@ -721,4 +730,16 @@ namespace kagome::host_api {
     SL_TRACE_FUNC_CALL(logger_, res, signature, msg, pubkey);
     return res;
   }
+
+  void CryptoExtension::reset() {
+    batch_verify_.reset();
+  }
+
+  runtime::WasmSize CryptoExtension::batchVerify(runtime::WasmSize ok) {
+    if (not batch_verify_) {
```

Comment:
I think that the spec says that if batch verification is not started, we should just return the result of the verification.